src/features/views/components/ViewColumnDialog/ColumnGallery/SearchResults.tsx

```diff
@@ -33,9 +33,9 @@ const SearchResults = ({
             <ColumnChoiceCard
               alreadyInView={alreadyInView}
               cardVisual={searchResult.renderCardVisual(
-                alreadyInView ? 'gray' : '#234890'
+                alreadyInView ? 'gray' : searchResult.color
               )}
-              color={alreadyInView ? 'gray' : '#234890'}
+              color={alreadyInView ? 'gray' : searchResult.color}
```

Comment:
This is my mistake, for not saying when asking you to do this: the "gray" should be replaced by "theme.palette.onSurface.disabled"